docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
+
+[Guzzle](http://docs.guzzlephp.org/en/latest/) is a library for accessing URL's through PHP's [cURL library](http://php.net/manual/book.curl.php). Guzzle is automatically installed with Codeception. You can easily access it from any test file:
+
+```php
+// ExternalCest.php
```

Comment:
Yes, please do. Our syntax on website is very sensitive to `<?php` header and won't highlight PHP code without it :upside_down_face: